Tools/machines/perlmutter-nersc/perlmutter_gpu.sbatch

```diff
@@ -24,9 +24,21 @@
 #SBATCH -o WarpX.o%j
 #SBATCH -e WarpX.e%j
 
-# executable & inputs file or python interpreter & PICMI script here
-EXE=./warpx
-INPUTS=inputs
+# PICMI Python script or executable?
+USE_PICMI=true
+
+if [[ "${USE_PICMI}" = true ]]
+then
+    EXE=python3
+    INPUTS=PICMI_script.py
+    #   for GPU-aware MPI in PICMI, set
+    #   Simulation(..., warpx_amrex_use_gpu_aware_mpi=True)
```

Comment:
Something I've thought about is allowing setting of input parameters from the command line when using Python. Something like
```python3 PICMI_script.py --inputs amrex.use_gpu_aware_mpi=1```
But I'm not sure it's a good idea. For one, it would conflict with any argument parsing that the input script might do.